package.json

```diff
@@ -40,6 +41,7 @@
     "build:full": "npm run clean:lerna && npm install && npm test",
     "pretest": "npm run clean && npm run build",
     "test": "node packages/build/bin/run-nyc npm run mocha --scripts-prepend-node-path",
+    "test:ci": "npm run pretest && npm run test -- --ignore-scripts",
```

Comment:
I find this confusing - how is `test:ci` different from `test`, when they both run `pretest` (build) and `test`?